spring-restdocs-asciidoctor-support/src/main/resources/extensions/operation_block_macro.rb

```diff
@@ -96,8 +96,8 @@ def write_content(content, snippet, operation)
     end
   end
 
-  def write_title(content, snippet, level, id, snippet_titles)
-    section_level = '=' * (level + 1)
+  def write_title(content, snippet, id, snippet_titles)
+    section_level = '=='
```

Comment:
Switch to hardcoding to two `=` feels like it may cause a regression to me. For example, will it not now be wrong if the macro is used in a deeply nested section?

Reply:
The new line 53 will adjust the level of all top level blocks the moment they are inserted. This avoid the warning that would otherwise occur when starting with a section of a too high level, and avoid having a regresseion.